Sources/Networking/CustomerAPI.swift

```diff
@@ -108,7 +112,7 @@ final class CustomerAPI {
                                                          presentedOfferingIdentifier: offeringIdentifier,
                                                          observerMode: observerMode,
                                                          initiationSource: initiationSource,
-                                                         subscriberAttributesByKey: subscriberAttributesByKey)
+                                                         subscriberAttributesByKey: subscriberAttributesToPost)
```

Comment:
I guess this would be null if the new mode is on... probably ok but I thought I would mention it.

Reply:
Yeah, since that's encoded in the type system, the method handles that explicitly. It's also tested now :)

Reply:
yeah I figured since the method took in a nullable value, it was safer to send in an null so that it wouldn't get translated into an empty list instead. I did test manually with POST receipt to make sure it worked fine